ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+function extract_oc(){
+    echo -e "Extracting oc\n"
+    local retry=5 
+    tmp_oc="/tmp/client"
+    mkdir -p ${tmp_oc}
+    while ! (env "NO_PROXY=*" "no_proxy=*" oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE});
+    do
+        echo >&2 "Failed to extract oc binary, retry..."
+        (( retry -= 1 ))
+        if (( retry < 0 )); then return 1; fi
+        sleep 60
+    done
+    mv ${tmp_oc}/oc /tmp -f
+    which oc
+    oc version --client
+}
+
+# Define the checkpoints/steps needed for the specific case
+function post-ocp-66839(){
+    if [[ "${BASELINE_CAPABILITY_SET}" != "None" ]]; then
```

Comment:
I am thinking if we should remove the condition, so that we can easily find out any mis-configuration issue in job config level.

Reply:
Prefer to keep it there to distinguish the case is skipped or failed.